jose-jwt/JWTSettings.cs

```diff
@@ -28,7 +28,7 @@ public class JwtSettings
                 { JwsAlgorithm.ES256, new EcdsaUsingSha(256) },
                 { JwsAlgorithm.ES384, new EcdsaUsingSha(384) },
                 { JwsAlgorithm.ES512, new EcdsaUsingSha(521) }
-#elif NETSTANDARD1_4 || NET461
+#elif NETSTANDARD || NET461

```

Comment:
What's difference between NETSTANDARD vs NETSTANDARD1_4?  I guess 1.4 was used to indicate minimal required version.  Will NETSTANDARD include versions below 1.4?

Reply:
Yes, NETSTANDARD will include versions below 1.4, but since we are only compiling to 1.4 and 2.1 as netstandard targetframeworks, it shouldn't have any effect.
These checks are just there to differentiate between code compatible with fullframework and .NET core